pkg/webhooks/mutation.go

```diff
@@ -68,7 +69,7 @@ func (ws *WebhookServer) HandleMutation(
 		}
 
 		if !engineResponse.IsSuccessful() && len(engineResponse.GetFailedRules()) > 0 {
-			logger.Info("failed to apply policy", "policy", policy.Name, "failed rules", engineResponse.GetFailedRules())
+			logger.Error(errors.New("some rules failed"), "failed to apply policy", "policy", policy.Name, "failed rules", engineResponse.GetFailedRules())
```

Comment:
Were you able to reproduce the issue? According to the description, Kyverno showed the policy application failed with an empty rule, do we know what was the issue exactly?
```
I0109 22:17:26.245363       1 mutation.go:66] WebhookServer "msg"="failed to apply policy"... "failed rules"=[""]...
```

Reply:
Yes, in fact, changes in pkg/engine/mutate/mutation.go are did the thing and we have got the failed rules list